client/app/states/orders/details/details.html

```diff
@@ -22,7 +22,7 @@
               </div>
               <div class="ss-details-header__title">
                 <h2>{{ ::vm.order.name }}</h2>
-                <h4>ordered {{ ::(vm.order.placed_at || vm.order.updated_at) | date }}</h4>
+                <h4> {{'ordered' | translate}} {{ ::(vm.order.placed_at || vm.order.updated_at) | date }}</h4>
```

Comment:
This is slightly problematic in that it assumes "ordered (on) 2017-12-15" is the same "ordered" as in the sentence "It's already ordered." Which it may not be. Also, some languages may prefer to say "2017-12-15 ordered".

According to https://angular-gettext.rocketeer.be/, looks like just moving the `translate` attribute on that `h4` should do the right thing.

(Essentially we want the equivalent of `sprintf(_("ordered %{on}"), { on: textNotSubjectToTranslation })` instead of `_("ordered") + textNotSubjectToTranslation`, so that the translator gets that extra context info.)

Reply:
@himdel thanks for the perspective!! Agree, the context of the word "ordered" matters. `<h4 translate>` will effectively put the translator in charge of modifying the rest of that html though, a situation I do not believe is ideal. Maybe this should be reworded all together?

Reply:
Aah, yeah, if that's a thing we don't want...

What about just adding a preposition? `{{"ordered on" | translate}}` gives more context. (But still can't be reordered.)

After that.. well we can always do `sprintf(__("ordered %s"), $tilter('date')(vm.order.placed_at || vm.order.updated_at))` in the controller..

Reply:
Cc @mzazrivec, if you have a preference here.. :)